client/lib/js/react/components/accounts.js

```diff
@@ -4,14 +4,15 @@ import { Row } from 'react-bootstrap/lib/';
 
 import classnames from 'classnames/';
 
-import AccountActions from '../../../lib/js/react/actions/account_actions';
-import AccountStore from '../../../lib/js/react/stores/account_store';
+import AccountActions from '../actions/account_actions';
+import AccountStore from '../stores/account_store';
 
-import AscribeSpinner from '../../../lib/js/react/components/spinner';
+import AscribeSpinner from './spinner';
 
 const Accounts = React.createClass({
```

Comment:
Might be good to rename it to `AccountList` and add a `className` prop to allow for easier restyling.